code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -49,7 +49,7 @@
 					new /obj/item/weapon/gun/projectile/pistol(get_turf(H))
 					new /obj/item/gun_part/silencer(get_turf(H))
 				if("cannon")
-					new /obj/item/weapon/gun/energy/lasercannon(get_turf(H))
+					new /obj/item/weapon/gun/energy/laser/cannon(get_turf(H))
```

Comment:
because before I needed to copypaste the /obj/item/weapon/gun/energy/laser effects to /obj/item/weapon/gun/energy/lasercannon which wasn't a subtype of /laser for some reason